### PR TITLE
[SSL Simulation Protocol] Added local velocity commands

### PR DESCRIPTION
**Pull request by jsn0716**
### Description
Since apparently move local velocity commands are the best supported, add the option to create robot command using local velocity

### Testing Done
Basic unit tests

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [x] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [x] **Remove all commented out code**
- [x] **Remove extra print statements**: for example, those just used for testing
- [x] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [x] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

**Review by jonathanlew — COMMENTED** [review]
Just curious why we need both `createMoveLocalVelocity` and `createRobotMoveCommand`.

File: src/software/proto/message_translation/ssl_simulation_robot_control.cpp [hunk 1]
```diff
@@ -1,14 +1,54 @@
...
+}
+
+// Converts rpm and wheel_radius [m] to speed [m/s]
+float rpm_to_m_per_s(float rpm, float wheel_radius)
+{
+    return (2 * (float)M_PI * rpm * wheel_radius) / 60.0f;
+}
+
+std::unique_ptr<SSLSimulationProto::MoveLocalVelocity> createMoveLocalVelocity(
+    double front_right, double front_left, double back_left, double back_right)
```
Comment (kelvinkoon) [inline src/software/proto/message_translation/ssl_simulation_robot_control.cpp]: Nit: Can we change `front_right` to something like `front_right_speed`? Upon first reading the code, I thought `front_right` was referring to the wheel itself (just something to denote it's the speed).

**Review by kelvinkoon — DISMISSED** [review]
LGTM, just one non-blocking

File: src/software/proto/message_translation/ssl_simulation_robot_control.cpp (hunk 1 above)
Comment (jsn0716) [reply src/software/proto/message_translation/ssl_simulation_robot_control.cpp]: Fixed

**Comment by jsn0716** [comment]
@jonathanlew we don't really need both, I just didn't bother removing it

**Review by jonathanlew — COMMENTED** [review]
When would we ever need to use `SSLSimulationProto::MoveLocalVelocity` though? It seems like we could just combine them, i.e. `std::unique_ptr<SSLSimulationProto::RobotMoveCommand> createRobotMoveCommand(   double wheel_rpm_front_right, double wheel_rpm_front_left, double wheel_rpm_back_left,
    double wheel_rpm_back_right)`

**Comment by jsn0716** [comment]
@jonathanlew Combined to `createRobotMoveCommand`

**Review by kelvinkoon — APPROVED** [review]

**Review by akhilveeraghanta — APPROVED** [review]

**Review by jonathanlew — APPROVED** [review]
:+1:

**Pull request merged**